pkg/types/validator.go

```diff
@@ -138,16 +148,36 @@ func (v ValidatorWithInfo) Serialize(disableEmojis bool) string {
 		}
 	}
 
+	// Format block hashes: show first 3 chars, "nil" for empty, "000" for nil block
+	prevoteHash := formatBlockHash(v.RoundVote.PrevoteBlockHash)
+	precommitHash := formatBlockHash(v.RoundVote.PrecommitBlockHash)
+
 	return fmt.Sprintf(
-		" %s %s %s %s%% %s ",
+		" %s%s %s%s %s %s%% %s ",
```

Comment:
let's maybe add another space between the emoji and the vote itself? otherwise it looks a bit ugly
<img width="1568" height="745" alt="image" src="https://github.com/user-attachments/assets/c52dbb0e-f5b1-45f4-8274-94e7aba2ffb8" />